avaframe/ana3AIMEC/ana3AIMEC.py

```diff
@@ -45,10 +45,20 @@ def readAIMECinputs(avalancheDir, dirName='com1DFA'):
 
     profileLayer = glob.glob(os.path.join(avalancheDir, 'Inputs', 'LINES',
                                           '*aimec*.shp'))
+    try:
+        message = 'There should be exactly one path_aimec.shp file containing the avalanche paths in ' + avalancheDir + '/Inputs/LINES/'
```

Comment:
```suggestion
        message = 'There should be exactly one path_aimec.shp file containing (multiple) avalanche paths in ' + avalancheDir + '/Inputs/LINES/'
```
Maybe? To make put more emphasis on that you can have multiple paths in one shape file?

Reply:
for aimec there should only be one path. For AB I agree that there can be multiple paths in the shp file

Reply:
ah ok, then `paths -> path`